lib/DDG/Spice/NobelPrize.pm

```diff
@@ -0,0 +1,43 @@
+package DDG::Spice::NobelPrize;
+
+# ABSTRACT: Write an abstract here
+
+# Start at http://docs.duckduckhack.com/walkthroughs/forum-lookup.html if you are new
+# to instant answer development
+
+use DDG::Spice;
+use Data::Dumper;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'http://api.nobelprize.org/v1/prize.json?' 
+                . '&year=$1' 
+                . '&category=$2';
+
+
+# Triggers - https://duck.co/duckduckhack/spice_triggers
+triggers startend => 'nobel';
```

Comment:
Not sure what you are suggesting here. If only we see a startend with `nobel` we can trigger. 
Given that ofcourse we have either a `year` or a `category` or both specified. 

Refining the trigger to only certain kinds of phrases, is that necessary? 
I mean i get that people can try varying strings but we can safely ignore the complications and *still* show the relevant data. Don't you think?